InterfaceParser/parsers/sdl_rpc_v1.py

```diff
@@ -0,0 +1,31 @@
+"""SDLRPCV1 parser.
+
+Contains parser for SDLRPCV1 XML format.
+
+"""
+from parsers.parse_error import ParseError
+from parsers.rpc_base import RPCBase
+
+
+class Parser(RPCBase):
+    """SDLRPCV1 parser."""
+
+    @property
+    def get_version(self):
+        return '1.0.0'
+
+    def _check_enum_name(self, enum):
```

Comment:
This method does not exist in the cores current version of this parser. Any particular reason this was added?

Reply:
Version and particular property/getter added to each parsed and generator.
To version the scripts such that the generator script requires the same or lesser version of parser script. To avoid issues if the parser script needs to fix a bug (and becomes, e.g. 1.0.1) and the generator script stays at 1.0.0. As long as the generator script is the same or greater major version, it should be parsable. This requires some level of backward compatibility. They have to be the same major version.
https://github.com/smartdevicelink/sdl_evolution/issues/853#issuecomment-549891532

Reply:
@o-mishch My comment was regarding the method `_check_enum_name`

Reply:
I removed _check_enum_name as you requested